circle.yml

```diff
@@ -0,0 +1,18 @@
+dependencies: 
+  cache_directories: 
+    - ~/.cache/pip
+    - ~/wheelhouse
+  post: 
+    - .circle/services.sh
+    - ".circle/compose.sh run wheezy /bin/true"
+    - ".circle/compose.sh run wheezy"
+  pre: 
+    - .circle/cache-permissions.sh
+
+machine: 
+  services: 
+    - docker
+
+test:
+  override:
+    - ".circle/compose.sh run wheezy bash -c 'cp /root/Gemfile* ./ && bundle exec rspec'"
```

Comment:
Btw it's already in there just have a look inside [compose.yml](compose.yml). This is required for spliting build and test stages, though it's not really important at all (I mean this cp string :) ).